config/config.go

```diff
@@ -53,8 +56,8 @@ type BlockManagerConfig struct {
 	MaxProofTime time.Duration `mapstructure:"max_proof_time"`
 	// BatchSubmitMaxTime is how long should block manager wait for before submitting batch
 	BatchSubmitMaxTime time.Duration `mapstructure:"batch_submit_max_time"`
-	// MaxBatchSkew is the number of batches which are waiting to be submitted. Block production will be paused if this limit is reached.
-	MaxBatchSkew uint64 `mapstructure:"max_supported_batch_skew"`
+	// MaxBlockSkew is the number of blocks which are waiting to be submitted. Block production will be paused if this limit is reached.
+	MaxBlockSkew uint64 `mapstructure:"max_supported_block_skew"`
```

Comment:
confusing naming as `max_supported_block_skew` is the max dymint support, not what the user chooses.

Reply:
param renamed